src/main/webapp/index.html

```diff
@@ -2,104 +2,21 @@
-        <button class="map-category-button" id="restaurants-button">Restaurants</button>
-        <button class="map-category-button" id="places-button">Places to Visit</button>
-        <button class="map-category-button" id="bars-and-clubs-button">Bars and Clubs</button>
-        <button class="map-category-button" id="study-button">Study Places</button>
-      </div>
-    </section>
-    <div class="wrapper">
-      <!-- The Recommendation Map -->
-      <div id="map"></div>  
-      <!-- View Recommendation Area -->
-      <div id="rec-container">
-        <p id="category-header">- Places to Visit -</p>
-        <h3>Sky Garden</h3>
-        <h4>Location / Address</h4>
-        <p id="rec-address">51.510881° N, -0.083751° W</p>
-        <h4>Description</h4>
-        <p>Sky Garden is a great place to visit, with really good
-          views of London. You can enjoy the 360 degree view
-          of the city - for free! They are public gardens and 
-          definitely worth a visit when you have the time.
-          Remember to keep socially distanced!
-        </p>
-      </div>
-    </div>
-    <!-- Popup for Adding Recommendation -->
-    <div class="popup" id="popup-add-recs">
-      <div class="overlay"></div>
-      <div class="popup-content">
-        <div class="close-button" onclick="togglePopup()">&times;</div>
-        <h2>Add Recommendation</h2>
-        <p>Please fill out this form to add a recommendation.</p>
-        <!-- Form for Adding Recommendation -->
-        <form>
-          <label for="place-name">Place Name:</label><br>
-          <textarea type="text" id="place-name" name="place-name" cols="40" rows="1" required></textarea><br>
-          <label for="location">Location/Address:</label><br>
-          <textarea type="text" id="location" name="location" cols="40" rows="1" required></textarea><br>
-          <!-- Radio Buttons for Type of Recommendation -->
-          <div class="radio-buttons">
-            <input type="radio" name="category-list" id="restaurants" value="restaurants" required>
-            <label for="restaurants">Restaurants</label>
-            <input type="radio" name="category-list" id="places-to-visit" value="places-to-visit">
-            <label for="places-to-visit">Places to Visit</label><br>
-            <input type="radio" name="category-list" id="bars-and-clubs" value="bars-and-clubs">
-            <label for="bars-and-clubs">Bars and Clubs</label>
-            <input type="radio" name="category-list" id="study-places" value="study-places">
-            <label for="study-places">Study Places</label>
-          </div>
-          <label for="description">Description:</label><br>
-          <textarea type="text" id="description" name="description" cols="40" rows="8" required></textarea><br>
-          <input type="submit" value="Submit">
-        </form>
       </div>
     </div>
-    <footer>
-      <p>Made by Lorna Armstrong and Rania Rusly</p>
-    </footer>
```

Comment:
Shall we have the footer here too?

Reply:
No, since this page is now the login page which just shows the login button- the page with just navy background

Reply:
Sounds good!